.azure/bicep/config/worker-app-settings.bicep

```diff
@@ -0,0 +1,58 @@
+param keyVaultName string
+param appInsightsName string
+param appName string
+param appSettings object = {}
+
+param workerStorageConnectionStringSecret string
+
+var secretRef = '@Microsoft.KeyVault(VaultName=${keyVaultName};SecretName='
+
+var sharedSettings = {
+  AzureWebJobsStorage: '${secretRef}${workerStorageConnectionStringSecret})'
+  // Log database settings
+  LOG_DB_DRIVERNAME: '${secretRef}LOG_DB_DRIVERNAME)'
+  LOG_DB_PORT: '${secretRef}LOG_DB_PORT)'
+  LOG_DB_HOST: '${secretRef}LOG_DB_HOST)'
+  LOG_DB_DATABASE: '${secretRef}LOG_DB_DATABASE)'
+  LOG_DB_USERNAME: '${secretRef}LOG_DB_USERNAME)'
+  LOG_DB_PASSWORD: '${secretRef}LOG_DB_PASSWORD)'
+  // Data source database settings
+  DATASOURCE_NAME: '${secretRef}DATASOURCE_NAME)'
+  DATASOURCE_DB_DRIVERNAME: '${secretRef}DATASOURCE_DB_DRIVERNAME)'
+  DATASOURCE_DB_PORT: '${secretRef}DATASOURCE_DB_PORT)'
+  DATASOURCE_DB_HOST: '${secretRef}DATASOURCE_DB_HOST)'
+  DATASOURCE_DB_USERNAME: '${secretRef}DATASOURCE_DB_USERNAME)'
+  DATASOURCE_DB_PASSWORD: '${secretRef}DATASOURCE_DB_PASSWORD)'
+  DATASOURCE_DB_SCHEMA: '${secretRef}DATASOURCE_DB_SCHEMA)'
```

Comment:
again, not sure all of these are secrets

and can they use `kebab-case` for the secret keys